persistence/gorm/gorm.go

```diff
@@ -175,14 +175,22 @@ func (s *storage) Get(r any, conds ...any) (err error) {
 	return
 }
 
-func (s *storage) List(r any, offset int, limit int, conds ...any) error {
+func (s *storage) List(r any, orderBy string, asc bool, offset int, limit int, conds ...any) error {
 	var query = s.db
+	var orderDirection = "asc"
 
 	if limit != -1 {
```

Comment:
Do we need some kind of escaping here, since `orderBy` is taken from the outside and could potentially be used to inject something? Is gorm takeing care of this?

Reply:
No, a really good point. Im thinking of creating a `validation` function to check against a whitelist including e.g. `id`, `name` etc.